### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by double22a**
Add results

### CODE REVIEW

**Comment by robin1001** [comment]
please fix the lint problem.

**Review by double22a — COMMENTED** [review]
remove spaces

File: examples/multi_cn/s0/README.md [hunk 1]
```diff
@@ -61,6 +61,57 @@ magic results
...
+|--------------------------|------|
+| attention decoder        | 4.45 |
+| ctc greedy search        | 5.73 |
+| ctc prefix beam search   | 5.73 |
+| attention rescoring      | 4.75 |
+
+thcs30 results
+
+| decoding mode/chunk size | 16   |
+|--------------------------|------|
```
Comment (pengzhendong) [inline examples/multi_cn/s0/README.md:100]: You should align the table.

File: examples/multi_cn/s0/README.md [hunk 2]
```diff
@@ -61,6 +61,57 @@ magic results
...
+| attention decoder        | 13.78 |
```
Comment (pengzhendong) [inline examples/multi_cn/s0/README.md:101]: You should align the tables.

**Review by pengzhendong — APPROVED** [review]

**Review by double22a — COMMENTED** [review]
align

**Pull request merged**